app/assets/stylesheets/govuk_publishing_components/components/_button.scss

```diff
@@ -38,6 +38,8 @@ $gem-quiet-button-hover-colour: darken($gem-quiet-button-colour, 5%);
     margin-bottom: 0;
   }
 
+  &:first-child {
+    margin-right: govuk-spacing(2);
```

Comment:
I think I understand this, but is there ever a situation where there might be a third button? Would a safer future proof option be to give all the buttons a left margin, but remove on the first one?